lib/mi_bridges/driver/others_types_of_income_page.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module MiBridges
+  class Driver
+    class OtherTypesofIncomePage < BasePage
+      TITLE = "Other Types of Income"
+
+      def setup; end
+
+      def fill_in_required_fields
+        # social security?
+        # foster care / adoption
```

Comment:
Style: code comments? Leftover todos?

Reply:
see PR commit msg for context on this. Think we need to page @alanjosephwilliams for a final verdict